ajax/libs/graphdracula/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "author": {
+    "name": "Johann Philipp Strathausen",
+    "email": "philipp@stratha.us",
+    "url": "http://stratha.us"
+  },
+  "name": "graphdracula",
+  "description": "JavaScript Graph Layouting and Drawing",
+  "version": "1.0.2",
+  "homepage": "http://www.graphdracula.net",
+  "license": "MIT",
+  "keywords": [
+    "bellman-ford",
+    "dijkstra",
+    "dracula",
+    "graph",
+    "math",
+    "raphael",
+    "snap",
+    "static",
+    "svg",
+    "vml"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/strathausen/dracula.git"
+  },
+  "dependencies": {
+    "lodash": "^4.14.1",
+    "uuid": "^2.0.2"
+  },
+  "optionalDependencies": {},
+  "autoupdate": {
```

Comment:
We prefer to use npm auto-update config if the library had been published on npm. Thanks. ref: [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md)